go/ocr2/decryptionplugin/decryption.go

```diff
@@ -79,7 +79,16 @@ func (dp *decryptionPlugin) Query(ctx context.Context, ts types.ReportTimestamp)
 	)
 
 	queryProto := Query{}
+	ciphertextIDs := make(map[string]bool)
 	for _, request := range decryptionRequests {
+		if _, ok := ciphertextIDs[string(request.CiphertextId)]; ok {
```

Comment:
no need to assign `ok` (the same in other places)

Reply:
Not sure I understand.

Reply:
I meant just to `if ciphertextIDs[string(request.CiphertextId)] {...`